pkg/semtech-loramac/patches/0002-adapt-for-RIOT.patch

```diff
@@ -0,0 +1,281 @@
+From 0e457af5dab43677f7ce048d21db3bcb85cf7b66 Mon Sep 17 00:00:00 2001
+From: Alexandre Abadie <alexandre.abadie@inria.fr>
+Date: Wed, 13 Dec 2017 17:43:00 +0100
+Subject: [PATCH] adapt for RIOT
+
+---
+ src/mac/LoRaMac.c                   | 32 +++++++++++++++-----------------
+ src/mac/LoRaMac.h                   |  4 +++-
+ src/mac/region/Region.c             |  2 +-
+ src/mac/region/RegionAS923.c        |  2 +-
+ src/mac/region/RegionAU915.c        |  2 +-
+ src/mac/region/RegionCN470.c        |  2 +-
+ src/mac/region/RegionCN779.c        |  2 +-
+ src/mac/region/RegionCommon.c       |  2 +-
+ src/mac/region/RegionEU433.c        |  2 +-
+ src/mac/region/RegionEU868.c        |  2 +-
+ src/mac/region/RegionIN865.c        |  2 +-
+ src/mac/region/RegionKR920.c        |  2 +-
+ src/mac/region/RegionUS915-Hybrid.c |  2 +-
+ src/mac/region/RegionUS915.c        |  2 +-
+ 14 files changed, 30 insertions(+), 30 deletions(-)
+
+diff --git a/src/mac/LoRaMac.c b/src/mac/LoRaMac.c
+index 0750ee5..1e2d6e6 100644
+--- a/src/mac/LoRaMac.c
++++ b/src/mac/LoRaMac.c
+@@ -17,7 +17,8 @@ License: Revised BSD License, see LICENSE.TXT file include in the project
+ 
+ Maintainer: Miguel Luis ( Semtech ), Gregory Cristian ( Semtech ) and Daniel Jaeckle ( STACKFORCE )
+ */
+-#include "board.h"
++#include "semtech-loramac/board.h"
++#include "utilities.h"
+ 
+ #include "LoRaMac.h"
+ #include "region/Region.h"
+@@ -316,11 +317,6 @@ static LoRaMacPrimitives_t *LoRaMacPrimitives;
+ static LoRaMacCallback_t *LoRaMacCallbacks;
+ 
+ /*!
+- * Radio events function pointer
+- */
+-static RadioEvents_t RadioEvents;
+-
+-/*!
+  * LoRaMac duty cycle delayed Tx timer
+  */
+ static TimerEvent_t TxDelayedTimer;
+@@ -687,8 +683,9 @@ static void PrepareRxDoneAbort( void )
+     LoRaMacFlags.Bits.MacDone = 1;
+ 
+     // Trig OnMacCheckTimerEvent call as soon as possible
+-    TimerSetValue( &MacStateCheckTimer, 1 );
+-    TimerStart( &MacStateCheckTimer );
++    // TimerSetValue( &MacStateCheckTimer, 1 );
++    // TimerStart( &MacStateCheckTimer );

```

Comment:
Rather remove instead of comment out, otherwise porting to later versions might more easily provoke conflicts in these lines.

Reply:
done